tools/tools.go

```diff
@@ -1,3 +1,6 @@
+//go:build codegen
+// +build codegen
```

Comment:
Not sure about those tags 
they will compile if you will run if used like this: 
`go run ./tools --tags=codegen`

what is the meaning of `codegen`? 
if i understand correctly you want to exclude this file from compiling for good, perups consider using someother term so it wont be confusing to when this file will be compiled.
 ```suggestion
//go:build never
// +build never
```
wdyt?

Reply:
@dasionov  I updated the build tag to `never` as suggested.

Reply:
Thanks. 
Please take a look at this [comment](https://github.com/kubevirt/kubevirt/pull/14721#issuecomment-3041676897)
since this workaround has a better handling in the new go version, so i believe we should wait for go 1.24 bump before implemanting this fix